src/runtime/typemanager.cs

```diff
@@ -434,7 +434,7 @@ internal static IntPtr BasicSubType(string name, IntPtr base_, Type impl)
             return type;
         }
 
-        enum TypeSlots
+        internal enum TypeSlots : long
```

Comment:
Actually, according to [C API docs](https://docs.python.org/3/c-api/type.html#c.PyType_Slot) this type is C's `int` type, which is 32 bit on [most systems](https://www.ibm.com/support/knowledgecenter/en/SSFKSJ_9.0.0/com.ibm.mq.ref.dev.doc/q104610_.htm).